docs/services/slam/_index.md

```diff
@@ -31,3 +31,203 @@ Integrated SLAM Libraries include:
 | Model | Description |
 | ----- | ----------- |
 | [`viam:slam:cartographer`](cartographer/) | [The Cartographer Project](https://github.com/cartographer-project) performs dense SLAM using LIDAR data. |
+
+## API
+
+The SLAM Service supports the following methods:
+
+Method Name | Description
+----------- | -----------
+[`GetPosition`](#getposition) | Get the current position of the specified source component in the point cloud SLAM map.
```

Comment:
[note] Once we support multiple sensor inputs, we'll want to come back and update this description and similar descriptions below.

cc @PStrutz , @jeremyrhyde. Also, for Patricia and Jeremy to think about - with IMU support, does it still make sense for GetPosition to return a single componentReference?